precompiles/xvm/evm_sdk/flipper.sol

```diff
@@ -2,17 +2,18 @@ pragma solidity ^0.8.0;
 
 interface XVM {
     function xvm_call(
-        uint8 calldata vm_id,
+        uint8 vm_id,
         bytes calldata to,
         bytes calldata input,
-    ) external;
+        uint256 value
+    ) external payable returns (bool success, bytes memory data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `xvm_call` function in the `XVM` interface has been modified to include a new parameter `value` of type `uint256`. This parameter represents the value to transfer during the call. The function now returns a tuple `(bool success, bytes memory data)`. Please ensure that this change is properly documented and communicated to users of the interface.



<!-- This is an auto-generated comment by OSS CodeRabbit -->